html_app/scb_ex/addmultipledialog2.soy

```diff
@@ -12,72 +12,62 @@
         Add Samples </h1>
 
     <div class='scb_ex_inner_dialog_body'>
-
-        <table class="scb_s_experiment_setup_table" role='grid' aria-label='Cell Lines'>
-
-
-
-//            {if $assignment.id == 'scb_ex2'}
-
-            <thead class="scb_s_experiment_setup_table_head">
-
-            {foreach $heading in $template.ui.add_multiple_dialog.headings}
-            <td role='columnheader' aria-label='{$heading}' class='scb_s_experiment_setup_table_heading'>
-                {$heading}
-            </td>
-            {/foreach}
+        <table class="scb_s_experiment_setup_table" role='grid' >
+            <thead class="scb_s_experiment_setup_table_head" >
+                {foreach $heading in $template.ui.add_multiple_dialog.headings}
+                <td role='columnheader' aria-label='{$heading}'
+                    class="scb_s_experiment_setup_table_heading
+                    {if $heading == 'Strain'} scb_s_experiment_setup_table_heading_strain{/if}
+                    {if $heading == ''} scb_s_experiment_setup_table_heading_checkbox{/if}" >
+                    {$heading}
+                </td>
+                {/foreach}
             </thead>
-
-            <tbody class="scb_s_experiment_setup_table_body">
-                {foreach $cell_line in $template.ui.add_multiple_dialog.order}
+            {foreach $cell_line in $template.ui.add_multiple_dialog.order}
+                <tbody class="scb_s_experiment_setup_table_body">
```

Comment:
are you sure this is right? you start `foreach` before `tbody` and you do not close after it.